packages/apps/spaces/app/organizations/[id]/components/Timeline/events/slack/SlackMessageCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { PropsWithChildren } from 'react';
+import { Card, CardBody } from '@ui/presentation/Card';
+import { Flex } from '@ui/layout/Flex';
+import { Text } from '@ui/typography/Text';
+import { Avatar } from '@ui/media/Avatar';
+import User from '@spaces/atoms/icons/User';
+import { ViewInSlackButton } from '@organization/components/Timeline/events/slack/ViewInSlackButton';
+
+interface SlackMessageCardProps extends PropsWithChildren {
+  name: string;
+  profilePhotoUrl?: null | string;
+  content: string;
+  onClick?: () => void;
+  date: string;
+  w?: any;
+  showDateOnHover?: boolean;
```

Comment:
You could extend this interface with FlexProps, or CardProps or what suits better. It'll make all css props available, especially if you spread the ...rest into a chakra component.